packages/bindings/src/types.rs

```diff
@@ -3,9 +3,9 @@ use cosmwasm_schema::cw_serde;
 #[cw_serde]
 pub struct BtcBlockHeader {
     pub version: i32,
-    // hex encoded hash of previous block
+    // btc compatible (serialized in reverse byte order) hex encoded hash of previous block
```

Comment:
Should we call it reverse byte order or little endian?

Reply:
rust bitcoin calls it `reverse byte order` in their docs about hashes :

```
For compatibility with the existing Bitcoin infrastructure and historical
and current versions of the Bitcoin Core software itself, this and
other [`sha256d::Hash`] types, are serialized in reverse
byte order when converted to a hex string via [`std::fmt::Display`] trait operations.
See [`hashes::Hash::DISPLAY_BACKWARD`] for more details.
```
So I would stick to that.